src/commands/start.ts

```diff
@@ -71,7 +71,16 @@ const setPluginConfig = (config: ApplicationConfig, flags: Flags): void => {
 export default class StartCommand extends Command {
 	static description = 'Start Lisk Core Node.';
 
-	static examples = ['start', 'start --network dev --data-path ./data --log debug'];
+	static examples = [
+		'start',
+		'start --network devnet --data-path ~/lisk-data --log debug',
```

Comment:
'start --network devnet --data-path /path/to/data-dir --log debug',
```suggestion
		'start --network devnet --data-path /path/to/data-dir --log debug',
```